### DISCUSSION TITLE
MeshCentral LDAP filter for active directory group.

### DISCUSSION THREAD

**Discussion by artem33region** [Ylianst/MeshCentral#4909] [Q&A]
Who can help me? I want to log in to meshcentral only for group members, but i can't find "searchfilter" expression for AD group.

My filter:
searchFilter": "(sAMAccountName={{username}})"

May be someone use filter for group?

**Comment by compumatter** [comment]
This post may be of value: https://github.com/Ylianst/MeshCentral/discussions/4956
